pom.xml

```diff
@@ -294,10 +269,12 @@
             <groupId>org.eclipse.microprofile.rest.client</groupId>
             <artifactId>microprofile-rest-client-api</artifactId>
         </dependency>
+<!-- TODO I'm assuming we don't need this since telemetry does not have an MP API
```

Comment:
yes

Reply:
Yes "we need it"? or "yes, it can be removed"?

Reply:
yes, it can be removed.